app/views/dashboard.py

```diff
@@ -33,11 +33,20 @@ class DashboardView(MobileSupportedView):
 
     def get(self, request):
         game = most_recent_game()
+        participant = request.user.participant(game)
```

Comment:
For some reason, this line returns `NoneType` for superuser accounts. We should investigate this later, but for now, it's causing line 43 to crash when a superuser logs in.